.github/workflows/ci-code.yml

```diff
@@ -5,38 +5,10 @@ on:
     branches-ignore: [gh-pages]
   pull_request:
     branches-ignore: [gh-pages]
+    paths-ignore: ['docs/**']
```

Comment:
How does this paths-ignore work? If no files are changed, the workflow does not even run, correct? And so the paths-ignore will ignore changes in those files. So this workflow will only not run, if there are only changes in the `docs` directory. Is that correct?

Reply:
Yep, it will only run if the PR includes at least one file outside the docs directory
See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths